doc/developer_guide/developers.rst

```diff
@@ -951,8 +951,61 @@ code (a KernelSchedule with all its children), these are:
 - `FortranWriter()` in `psyclone.psyir.backend.fortran`
 - `OpenCLWriter()` in `psyclone.psyir.backend.opencl`
 
-Additionally, there is a `psyclone.psyir.backend.c` back-end, but at the
-moment it is only capable of processing partial PSyIR expressions.
+Additionally, there are two partially implemented back-ends
+
+- `psyclone.psyir.backend.c` which is currently limited to processing
+  partial PSyIR expressions.
+- `SIRWriter()` in `psyclone.psyir.backend.sir` which can generate
+  valid SIR from simple Fortran code conforming to the NEMO API.
+
+SIR back-end
+============
+
+The SIR back-end is limited in a number of ways:
+
+- Fortran code containing 3 dimensional directly addressed arrays, with simple
+  stencil accesses, iterated with triply nested loops. Imperfectly
+  nested loops, doubly nested loops, etc will cause a VisitorError
+  exception.
+- anything other than real arrays (integer, logical etc.) will cause
+  incorrect SIR code to be produced (see issue #468).
+- calls are not supported (and will cause a VisitorError exception).
+- control logic, other than triply nested loops, (if, case etc), is
+  not supported (and will cause a VisitorError exception).
+
+The current implementation is not able to deal with variables local to
+a region (which the SIR expects), as, in Fortran, the standard scope
+of a local variable is the whole routine, not a sub-region of code.
+
+The current implementation also outputs text rather than running dawn
+directly. This text needs to be pasted into another script in order to
+run dawn, see the NEMO API example 4.
```

Comment:
Added using ref rather than xref. It's not showing up at the moment but I think that is because the label needs to be published in the user_guide first.

Reply:
Yes, sorry. I was using "X-ref" as an abbreviation for cross reference.